com.github.jms55.Sandbox.json

```diff
@@ -0,0 +1,45 @@
+{
+    "app-id": "com.github.jms55.Sandbox",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "finish-args": [
```

Comment:
You might want `--filesystem=xdg-videos` to be able to save the recorder videos

Reply:
Does `--filesystem=xdg-videos/sandbox` work, if I only write to a specific subfolder?

Reply:
Should work fine

Reply:
This did not work actually, only xdg-videos without a subpath worked.